vhdbuilder/packer/init-variables.sh

```diff
@@ -276,7 +276,7 @@ if [ "$OS_TYPE" == "Windows" ]; then
 			os_disk_size_gb=${WINDOWS_2019_CONTAINERD_OS_DISK_SIZE_GB}
 		fi
 		;;
-	"2022-containerd")
+	"2022-containerd" | "2022-containerd-gen2")
```

Comment:
For Gen 1 and Gen 2, they both go into the same code block here. Initially, the WINDOWS_SKUs are the same "2022-containerd". Adding "gen2" suffix to differentiate??